service-discovery/README.adoc

```diff
@@ -101,28 +103,56 @@ Before you can bind Kafka instances in {product-long} to applications in OpenShi
 . Click the perspective switcher in the upper-left corner. Switch to the *Administrator* perspective.
 . In the left menu, click *Operators* > *OperatorHub*.
 . In the *Filter by keyword* field, enter `RHOAS`.
-. Select the *OpenShift Application Services (RHOAS)* Operator.
+. In the filtered results, select the *OpenShift Application Services (RHOAS)* Operator.
 . If you see a dialog box entitled *Show community Operator*, review the included information. When you've finished, click *Continue*.
 +
 An information sidebar for the RHOAS Operator opens.
 
 . In the sidebar, review the information about the RHOAS Operator and click *Install*.
 . On the *Install Operator* page, perform the following actions:
 .. For the *Installation mode* option, ensure that `All namespaces on the cluster` is selected.
-.. For the *Update channel* and *Update approval* options, keep the default values.
+.. For the *Update channel*, *Installed Namespace*, and *Update approval* options, keep the default values.
 .. Click *Install*.
 . When the installation process finishes, click *View Operator* to see the RHOAS Operator details. Alternatively, in the left menu, click *Operators* > *Installed Operators* and then click the RHOAS Operator.
 +
 Observe that the RHOAS Operator is installed in the `openshift-operators` namespace by default.
 
+[id="proc-installing-service-binding-operator_{context}"]
+== Installing the Service Binding Operator on OpenShift
```

Comment:
This guide initially used SDK for binding to minimize the number of steps and complexity.